components/editor/ui/SaveEditorContentButton.tsx

```diff
@@ -78,9 +84,15 @@ const SaveEditorContentButton = (props: editorProps) => {
               <TemporarySaveButton type="button" onClick={handleDraftSaveButton}>
                 임시저장
               </TemporarySaveButton>
-              <SaveButton type="button" onClick={handleOnClickArticlePublish}>
-                발행하기
-              </SaveButton>
+              {currentState === 'edit' ? (
+                <SaveButton type="button" onClick={handleOnClickArticlePublish}>
+                  수정하기
+                </SaveButton>
+              ) : (
+                <SaveButton type="button" onClick={handleOnClickArticlePublish}>
+                  발행하기
+                </SaveButton>
+              )}
```

Comment:
들어가는 텍스트만 다른거라면!
```suggestion
                <SaveButton type="button" onClick={handleOnClickArticlePublish}>
                  {currentState === 'edit' ? '수정하기' : '발행하기'}
                </SaveButton>
```
요로코롬 어떠신지!